.github/workflows/make-gen-delta.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Running make gen
         run: |
           make gen
+      - name: Running go mod tidy
+        run: |
+          go mod tidy
```

Comment:
This should probably happen right before `make tools`. Which is actually a reason to do it here instead of as a part of the `gen` target.